src/jit/gentree.h

```diff
@@ -4887,6 +4899,94 @@ struct GenTreePutArgStk : public GenTreeUnOp
 #endif
 };
 
+#ifdef _TARGET_ARM_
+// Represent the struct argument: value is splitted in register(s) and stack
+struct GenTreePutArgSplit : public GenTreePutArgStk
+{
+    unsigned gtNumRegs;
+
+    GenTreePutArgSplit(GenTreePtr op1,
+                       unsigned slotNum PUT_STRUCT_ARG_STK_ONLY_ARG(unsigned numSlots),
+                       unsigned     numRegs,
+                       bool         isHfa,
+                       bool         putIncomingArgArea = false,
+                       GenTreeCall* callNode           = nullptr)
+        : GenTreePutArgStk(GT_PUTARG_SPLIT,
+                           TYP_STRUCT,
+                           op1,
+                           slotNum PUT_STRUCT_ARG_STK_ONLY_ARG(numSlots),
+                           putIncomingArgArea,
+                           callNode)
+        , gtNumRegs(numRegs)
+    {
+#ifdef FEATURE_PUT_STRUCT_ARG_STK
+        gtIsHfa = isHfa;
+#endif
+    }
+
+    // HFA args is not yet handled
+    regNumber gtOtherRegs[MAX_REG_ARG - 1];
+
```

Comment:
Can you add a small comment here about what this is, and its relationship to gtRegNum?